pkg/apis/core/types.go

```diff
@@ -2430,6 +2461,14 @@ type ContainerStatus struct {
 	// +optional
 	ContainerID string
 	Started     *bool
+	// Node compute resources allocated for the container.
+	// +featureGate=InPlacePodVerticalScaling
+	// +optional
+	ResourcesAllocated ResourceList
+	// Compute resource requests and limits enacted on the running container.
```

Comment:
You might add some text to make clear why there are two seemingly-same fields and what the differences are?

Reply:
How does this look? 
```diff
diff --git a/pkg/apis/core/types.go b/pkg/apis/core/types.go
index a78ba779c90..6cd145e6bac 100644
--- a/pkg/apis/core/types.go
+++ b/pkg/apis/core/types.go
@@ -2461,11 +2461,14 @@ type ContainerStatus struct {
        // +optional
        ContainerID string
        Started     *bool
-       // Node compute resources allocated for the container.
+       // ResourcesAllocated represents the compute resources allocated for this container by the
+       // node. Kubelet sets this value to Container.Resources.Requests upon succesful pod admission
+       // and after successfully admitting desired pod resize.
        // +featureGate=InPlacePodVerticalScaling
        // +optional
        ResourcesAllocated ResourceList
-       // Compute resource requests and limits enacted on the running container.
+       // Resources represent the compute resource requests and limits that have been successfully
+       // enacted on the running container after it has been started or has been successfully resized.
        // +featureGate=InPlacePodVerticalScaling
        // +optional
        Resources *ResourceRequirements
```

Reply:
ACK